src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -493,12 +513,43 @@ protected override void FilterDiagnostics(AnalysisData analysisData, Func<Diagno
 
         private class IDELatestDiagnosticGetter : LatestDiagnosticsGetter
         {
-            public IDELatestDiagnosticGetter(DiagnosticIncrementalAnalyzer owner) : this(owner, null)
+            private readonly bool _concurrent;
```

Comment:
I don't know what that implies for other diagnostic getters. For now adding this support only for FixAll path, we can add support for other getters when needed.